server/pbenchinacan/load_keycloak.sh

```diff
@@ -44,15 +76,20 @@ echo
 echo "Creating ${REALM} realm"
 echo "--------------"
 
-curl -i -X POST "${KEYCLOAK_HOST_PORT}/admin/realms" \
+status_code=$(curl -f -i -o /dev/null -w "%{http_code}" -X POST "${KEYCLOAK_HOST_PORT}/admin/realms" \
   -H "Authorization: Bearer ${ADMIN_TOKEN}" \
   -H "Content-Type: application/json" \
-  -d '{"realm": "'${REALM}'", "enabled": true}'
+  -d '{"realm": "'${REALM}'", "enabled": true}')
+
+if [[ "${status_code}" != "201" ]]; then
+  echo ${status_code}
+  exit 0
```

Comment:
Why do we exit with zero?  That's typically a "success" status, but this looks like a failure case...yes?  Ditto lines 120 and 151.

Reply:
Those pesky little "off by one" errors ...